src/themes.json

```diff
@@ -603,6 +603,18 @@
 			}
 		],
 		"keywords": ["editor"]
+	},
+		{
+		"name": "SteamOS",
+		"repo": "https://github.com/rose-pine/SteamOS",
+		"maintainers": [
+			{
+				"name": "CarlTheObeseCat",
+				"url": "https://github.com/CarlTheObeseCat"
+			}
+		],
+		"keywords": ["OS"],
+		"variants": false
```

Comment:
This should be above sublime text as far as ordering :)

Reply:
Done